.github/workflows/rules/md101.js

```diff
@@ -0,0 +1,85 @@
+const { InlineTokenChildren } = require("./common/inlineTokenChildren");
+const { WordPattern } = require("./common/wordPattern");
+
+const keywords = [
+    new WordPattern("curl"),
+    new WordPattern("wget"),
+    new WordPattern("crontab"),
+    new WordPattern("cron"),
+    new WordPattern("netcat"),
+    new WordPattern("ping"),
+    new WordPattern("traceroute"),
+    new WordPattern("sudo"),
+    new WordPattern("(?<!(system |ISRG ))root(?! ca)", { suggestion: "root" }),// match "root", but not "root CA", "MacOS System Root" and "ISRG Root X1"
+    new WordPattern("true"),
+    new WordPattern("false"),
+    new WordPattern("jps"),
+    new WordPattern("name=value"),
+    new WordPattern("key=value"),
+    new WordPattern("time:value"),
+    new WordPattern("atsd.log"),
+    new WordPattern("start.log"),
+    new WordPattern("logback.xml"),
+    new WordPattern("graphite.conf"),
+    new WordPattern("command_malformed.log"),
+    new WordPattern("stdout"),
+    new WordPattern("stderr"),
+    new WordPattern("SIGTERM"),
+    new WordPattern("NaN"),
+    new WordPattern(".png", { noWordBoundary: true }),
+    new WordPattern(".xml", { noWordBoundary: true }),
+    new WordPattern(".jar", { noWordBoundary: true }),
+    new WordPattern(".gz", { noWordBoundary: true }),
+    new WordPattern(".tar.gz", { noWordBoundary: true }),
+    new WordPattern(".zip", { noWordBoundary: true }),
+    new WordPattern(".txt", { noWordBoundary: true }),
+    new WordPattern(".csv", { noWordBoundary: true }),
+    new WordPattern(".json", { noWordBoundary: true }),
+    new WordPattern(".pdf", { noWordBoundary: true }),
+    new WordPattern(".html", { noWordBoundary: true })
```

Comment:
```suggestion
    new WordPattern(".html", { noWordBoundary: true }),
    new WordPattern(".c", { noWordBoundary: true }),
    new WordPattern(".cpp", { noWordBoundary: true }),
    new WordPattern(".py", { noWordBoundary: true }),
    new WordPattern(".d", { noWordBoundary: true }),
    new WordPattern(".java", { noWordBoundary: true }),
    new WordPattern(".go", { noWordBoundary: true }),
    new WordPattern(".o", { noWordBoundary: true }),
    new WordPattern(".a", { noWordBoundary: true }),
    new WordPattern(".so", { noWordBoundary: true })
```
And any other language or file format we use.